labrad/server.py

```diff
@@ -352,6 +363,17 @@ def startup(self, protocol):
             self.disconnect(e)
             raise
 
+    @property
+    def client(self):
```

Comment:
I think a brief explanation here might help future readability, especially since this is critical to understanding the reuse trick.